spaceship/lib/spaceship/connect_api/models/profile.rb

```diff
@@ -70,7 +71,7 @@ def valid?
       # API
       #
 
-      def self.all(client: nil, filter: {}, includes: nil, fields: nil, limit: nil, sort: nil)
+      def self.all(client: nil, filter: {}, includes: nil, fields: nil, limit: Spaceship::ConnectAPI::MAX_OBJECTS_PER_PAGE_LIMIT, sort: nil)
```

Comment:
Couldn't this potentially silently break the behavior of existing _fastlane_ users that are using this method standalone (and not as part of _match_, for instance) and are relying on the fact that this will in fact return `all` the profiles? In fact this makes me think that this method was probably named `all` on purpose 😅 WDYT?

This same comment applies to all the `all` methods you modified the default value of the `limit` argument 🙇

Reply:
This will return `all` profiles. This `limit` is `The maximum number of resources to return per page, from 0 to 200` [API Docs](https://developer.apple.com/documentation/appstoreconnectapi/paginginformation/paging).

The default value is 20, and retrieving large lists requires more requests: 100 profiles = 5 requests. Since we don't fetch data dynamically and go through every page - the more we fetch in one batch, the better.

Reply:
Makes sense!! 💯 👏 
Thanks for clarifying!